samples/stock/demo/drag-on-axis/demo.js

```diff
@@ -0,0 +1,131 @@
+(function (Highcharts) {
+    // Configurable
+    const MOUSE_SENSITIVITY = 0.002,
+        SCROLL_SENSITIVITY = 0.0002;
+    function centerZoom(axis, strength) {
+        let zoomOffset = (axis.max - axis.min) / 2;
+        const center = axis.min + zoomOffset;
+
+        zoomOffset += strength * axis.toValue(1);
+
+        if (zoomOffset < 0) {
+            zoomOffset = 0;
+        }
+
+        axis.setExtremes(center - zoomOffset, center + zoomOffset, true, false);
+        if (axis.chart.resetZoomButton) {
+            axis.chart.resetZoomButton = axis.chart.resetZoomButton.destroy();
+        }
+        axis.chart.showResetZoom();
+    }
+
+    Highcharts.addEvent(Highcharts.Axis, 'afterInit', function () {
+        const axis = this,
+            renderer = axis.chart.renderer;
+
+        if (!axis.isXAxis && !axis.options.isInternal) {
+            // Create a transparent rectangle on yAxis to create a zoom area
+            axis.axisZoomRect = renderer.rect()
+                .attr({
+                    width: 50,
+                    height: 300,
+                    fill: 'transparent',
+                    zIndex: 8
+                })
+                .addClass('highcharts-no-mousewheel')
+                .css({ cursor: 'ns-resize' })
+                .add();
+
+            // Enabled zooming via mousewheel scroll
+            axis.axisZoomRect.on('mousewheel', event => {
```

Comment:
We still should prevent default action, when mousewheel and zooming occurs here. Simple `event.preventDefault()` should be fine.